aspnetcore/signalr/scale.md

```diff
@@ -104,14 +104,85 @@ The preceding conditions make it likely to hit the 10 connection limit on a clie
 
 ## Linux with Nginx
 
-Set the proxy's `Connection` and `Upgrade` headers to the following for SignalR WebSockets:
+The following contains the minimum required settings to enable WebSockets, ServerSentEvents, and LongPolling for SignalR:
 
 ```nginx
-proxy_set_header Upgrade $http_upgrade;
-proxy_set_header Connection $connection_upgrade;
+http {
+  map $http_connection $connection_upgrade {
+  "~*Upgrade" Upgrade;
```

Comment:
Out of curiosity, does the following work?

```suggestion
  "~*Upgrade" $http_connection;
```

Reply:
It does